patch qnn silu lowering

cccclai opened the pull request:
Summary: There are some edge lowering case missing. Change the logic to look for the op first because it's not decomposed

Differential Revision: D69636086

haowhsu-quic commented on backends/qualcomm/_passes/decompose_silu.py:
```diff
@@ -22,24 +22,22 @@ def _copy_meta(self, meta: Dict):
-                silu_node = src_partition.nodes[0]
-                with graph_module.graph.inserting_after(inputs[0]):
+        for node in graph.nodes:
+            if node.op == "call_function" and node.target in [
```
Maybe we can use set here `{`.

cccclai replied:
oh actually it's just one op and we can just compare...

haowhsu-quic reviewed (approved):
Thank you!

winskuo-quic reviewed (approved):
LGTM. Thanks.
Seems like there are lint issues.

cccclai committed:
patch qnn silu lowering (#8494)

Summary:

There are some edge lowering case missing. Change the logic to look for the op first because it's not decomposed

Reviewed By: billmguo

Differential Revision: D69636086
